quickstart/docker/docker-compose.yml

```diff
@@ -53,6 +54,7 @@ services:
       - ZITI_EDGE_CONTROLLER_RAWNAME="${ZITI_EDGE_CONTROLLER_RAWNAME}"
       - ZITI_EDGE_ROUTER_RAWNAME=ziti-edge-router
       - ZITI_EDGE_ROUTER_ROLES=public
+      - ZITI_EDGE_ROUTER_IDENTITY_ROLES=ziti.rest.host
```

Comment:
i like the ability to add roles this way, but doesn't this belong in the .env file?